sstables_loader.cc

```diff
@@ -678,11 +679,10 @@ future<> sstables_loader::download_task_impl::run() {
             } catch (...) {
             }
         });
+        co_await _progress_per_shard.start();
```

Comment:
If a user requests task status between the moment a task is registered in task manager and this point, we will do `map_reduce` on uninitialized `_progress_per_shard` (line 641)

Reply:
because neither `sharded<>` nor `task::impl` provides the facility to track if the progress is started, i added an enum to track the status of the progress.